EnglishFilter/sections/general_extensions.txt

```diff
@@ -46,10 +46,10 @@ pornxday.com,putlockerslinks.com,unblockninja.blue,baixarhentai.net,freeservice.
 pornyhd.com,pornxday.com,putlockerslinks.com,unblockninja.blue,baixarhentai.net,freeservice.info,branitube.net,hindimean.com,sports-stream.net,btdb.nl,drobfilx.com,22pixx.xyz,nacastle.com,eurostreaming.pink,imgbaron.com,embed.media,feurl.com,zidiplay.com,animeshouse.biz,erai-ddl3.info,erai-ddl2.info,erai-ddl.info,hitbits.io,scnlog.me,plytv.me,premiersport.tv,mixdrop.to,streamcdn.to,300mbfilms.io,jkanime.net,lovetheboxx.be,uii.io,mixdrop.co,filma24.cc,anime-odcinki.pl,phim2online.com,upzone.cc,likn.xyz,t0r.xyz,sportstream.to,acefile.co,watchxxxfreeinhd.com,haloani.ru,premiumleecher.com,short.pe,fjav.net,kastream.biz,dropapk.to,aflamfree.top,khatrimazafull.to,mirrorace.com,kuntv.pw,egybest2.com,javplay.me,imgoutlet.pw,ytmp3.eu#%#//scriptlet("abort-on-property-read", "popjs")
 !
 ! PopAds-b
-javhiv.com,nyafilmer.com,gotgayporn.com,letwatch.to,hqq.tv,xonline.vip,pornhd.com,thevideo.me,sextfun.com,vidup.me,imgoutlet.com,imgrock.net#%#Object.defineProperty(window, 'popns', { get: function() { return {}; } });
+nyafilmer1.com,javhiv.com,nyafilmer.com,gotgayporn.com,letwatch.to,hqq.tv,xonline.vip,pornhd.com,thevideo.me,sextfun.com,vidup.me,imgoutlet.com,imgrock.net#%#Object.defineProperty(window, 'popns', { get: function() { return {}; } });
 myreadingmanga.info,aflamyz.com,porntrex.com,faselhd.co,tnaflix.com,anysex.com,filecrypt.cc,icdrama.se,faselhd.com,jawcloud.co,xmoviesforyou.com,123moviesonline.stream,swapsmut.com,imgrock.info,alantv.com,publicleech.xyz,pornwhite.com,dir50.com,vshare.eu#%#Object.defineProperty(window, 'popns', { get: function() { return {}; } });
 czxxx.org,ashemaletube.com,fiuxy.eu,moviesand.com#%#AG_abortOnPropertyRead('popns');
-redhdtube.xxx,nyafilmer.*#%#//scriptlet('abort-on-property-read', 'popns')
+nyafilmer1.com,redhdtube.xxx,nyafilmer.*#%#//scriptlet('abort-on-property-read', 'popns')
```

Comment:
I don't need these rules.

Reply:
I added it just for in case. As I recall back, this site changes code and domains very frequently.

Reply:
OMG, All this happened while I was gone. Issues surpassed 400.

Reply:
> I added it just for in case. As I recall back, this site changes code and domains very frequently.

It is better to add the domain to the rule when it is really necessary.